CS2-CustomVotes/Services/CustomVoteApi.cs

```diff
@@ -34,8 +34,19 @@ public void AddCustomVote(string name, List<string> aliases, string description,
         _voteManager.AddVote(name, aliases, description, defaultOption, timeToVote, options, style, minVotePercentage);
     }
 
+    public void AddCustomVote(string name, List<string> aliases, string description, string defaultOption, float timeToVote,
+        Dictionary<string, VoteOption> options, string style, int minVotePercentage)
+    {
+        _voteManager.AddVote(name, aliases, description, defaultOption, timeToVote, options, style, minVotePercentage);
+    }
+
+    public void StartCustomVote(CCSPlayerController? player, string name, out string baseName)
```

Comment:
If you're exposing StartCustomVote you should probably also expose some EndCustomVote function just in case